### PR TITLE
Data processing code for B1+ efficiency

### PR DESCRIPTION
**Pull request by Kyota-exe**
In this PR, I will write all the processing/plotting code to generate the figures related to B1+ efficiency mentioned in #7.
The current `data_processing.ipynb` is a work in progress, and only generates for now a graph of the standard deviation of B1+ efficiency, which will soon be replaced by a graph of the inter-quartile range (IQR) across slices.

### CODE REVIEW

File: data_processing.ipynb [hunk 1]
```diff
@@ -0,0 +1,291 @@
+{
```
Comment (evaalonsoortiz) [inline data_processing.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    # Register TFL flip angle maps to the MP2RAGE UNI1 scan ⏳
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spinal-cord-7t/coil-qc-code/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Review by jcohenadad — COMMENTED** [review]
You need to add a cell that downloads the data, otherwise we are not able to test your code

**Comment by evaalonsoortiz** [comment]
I just ran the notebook locally. This is a good start. I added an additional figure (which I'll refer to as Fig1 here) which I think is helpful for visual assessment of results:

Fig1
image

The SC mask does not span the same levels for all subjects, as you can see in the Fig2 above. This means that for the IQR plot (see below) isn't really a fair comparison. 

Fig2
image

We should restrict the analysis to the same SC levels for all subjects. For this we will need a vertebral levels file.

**Comment by evaalonsoortiz** [comment]
I propose that we deal with the SC labels in another issue/PR and move forward with merging this one.

**Comment by jcohenadad** [comment]
That PR was merged with the notebook that included the results and the figures. As I mentioned in https://github.com/spinal-cord-7t/coil-qc-code/issues/5, binaries and any notebook outputs should _not_ be added to the git history. It will make cloning of the repository slower, and the repository will increase in size as new binaries are pushed to the repos.

**Comment by evaalonsoortiz** [comment]
> That PR was merged with the notebook that included the results and the figures. As I mentioned in #5, binaries and any notebook outputs should _not_ be added to the git history. It will make cloning of the repository slower, and the repository will increase in size as new binaries are pushed to the repos.

Fixed

**Pull request merged**